.github/CODEOWNERS

```diff
@@ -54,3 +54,4 @@ components/brave_referrals/browser/brave_referrals_service* @simonhong
 components/brave_sync/ @darkdh
 components/sync/driver/ @darkdh
 chromium_src/components/sync/ @darkdh
+chromium_src/components/sync_device_info/ @darkdh
```

Comment:
is all the code here critical and needs special review? If not please remove, otherwise please list individual files here